build.sbt

```diff
@@ -51,8 +51,8 @@ lazy val core = project
       Cats.effect,
       machinist,
       `slf4j-api`,
+      logback,
```

Comment:
so logbook is now required for all? would we have any problems with that?

Reply:
I don't expect any issues as we're already using logback as implementation for slf4j